src/components/OSCALSystemImplementationComponents.test.js

```diff
@@ -47,8 +47,17 @@ describe("OSCALSystemImplementationComponents", () => {
         parties={responsiblePartiesTestData}
       />
     );
-    const component = screen.getByText("Example Component").closest("tr");
-    const result = within(component).getByText("operational");
+    // Grab example party row beneath heading row
+    const exampleResponsiblePartiesRow = screen.getAllByRole("row")[1];
```

Comment:
Hmm `[1]` feels like we're just doing the same thing in a different way. Can you explain why this is correct? I don't think it's wrong; it's just not immediately obvious to me why it's right.

Reply:
This is the part I was mentioning I don't like very much.  Following the previous written tests, we're trying to ensure that the example content appears in the same row. There is nothing unique separating the rows that appear on screen, so they need to both be acquired when using the testing library. In this array, the first row is the heading cells, and the second is the example component.